### ISSUE TITLE
BackupTest.testGracefulShutdown3 

### ISSUE THREAD

**Issue by hasancelik** [hazelcast/hazelcast#6656]
```
java.lang.AssertionError: expected:<50000> but was:<49269>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at org.junit.Assert.assertEquals(Assert.java:631)
	at com.hazelcast.map.BackupTest$1.run(BackupTest.java:71)
	at com.hazelcast.test.HazelcastTestSupport.assertTrueEventually(HazelcastTestSupport.java:825)
	at com.hazelcast.test.HazelcastTestSupport.assertTrueEventually(HazelcastTestSupport.java:839)
	at com.hazelcast.map.BackupTest.checkSizeEventually(BackupTest.java:68)
	at com.hazelcast.map.BackupTest.testGracefulShutdown3(BackupTest.java:230)
```

https://hazelcast-l337.ci.cloudbees.com/job/Hazelcast-3.x-nightly/com.hazelcast$hazelcast/762/testReport/junit/com.hazelcast.map/BackupTest/testGracefulShutdown3/

**Comment by hasancelik** [comment]
https://hazelcast-l337.ci.cloudbees.com/job/Hazelcast-3.x-nightly/com.hazelcast$hazelcast/763/testReport/com.hazelcast.map/BackupTest/testGracefulShutdown3/

**Comment by hasancelik** [comment]
```
java.lang.AssertionError: expected:<50000> but was:<49651>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at org.junit.Assert.assertEquals(Assert.java:631)
	at com.hazelcast.map.BackupTest$1.run(BackupTest.java:71)
	at com.hazelcast.test.HazelcastTestSupport.assertTrueEventually(HazelcastTestSupport.java:851)
	at com.hazelcast.test.HazelcastTestSupport.assertTrueEventually(HazelcastTestSupport.java:865)
	at com.hazelcast.map.BackupTest.checkSizeEventually(BackupTest.java:68)
	at com.hazelcast.map.BackupTest.testGracefulShutdown3(BackupTest.java:238)
```

https://hazelcast-l337.ci.cloudbees.com/view/Official%20Builds/job/Hazelcast-3.x-sonar/com.hazelcast$hazelcast/962/testReport/junit/com.hazelcast.map/BackupTest/testGracefulShutdown3/

**Comment by hasancelik** [comment]
**maintenance**
```
java.lang.AssertionError: expected:<50000> but was:<46149>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at org.junit.Assert.assertEquals(Assert.java:631)
	at com.hazelcast.map.BackupTest$1.run(BackupTest.java:71)
	at com.hazelcast.test.HazelcastTestSupport.assertTrueEventually(HazelcastTestSupport.java:851)
	at com.hazelcast.test.HazelcastTestSupport.assertTrueEventually(HazelcastTestSupport.java:865)
	at com.hazelcast.map.BackupTest.checkSizeEventually(BackupTest.java:68)
	at com.hazelcast.map.BackupTest.testGracefulShutdown3(BackupTest.java:260)
```

https://hazelcast-l337.ci.cloudbees.com/view/Official%20Builds/job/Hazelcast-3.maintenance-nightly/com.hazelcast$hazelcast/894/testReport/junit/com.hazelcast.map/BackupTest/testGracefulShutdown3/

**Comment by enesakar** [comment]
fixed in 3.7. will not be fixed in 3.6 branch.
